cli_tools/import_precheck/check_sha2.go

```diff
@@ -36,7 +37,8 @@ func (s *sha2DriverSigningCheck) run() (*report, error) {
 		return r, nil
 	}
 
-	pkgs, err := packages.GetInstalledPackages()
```

Comment:
The new OSConfig packages changed the interface so I have to add this line, otherwise it can't pass the build